src/Geta.SEO.Sitemaps/XML/SitemapXmlGenerator.cs

```diff
@@ -352,8 +352,9 @@ protected virtual XElement GenerateSiteElement(IContent contentData, string url)
             {
                 var localeContent = contentData as ILocale;
                 var language = localeContent != null ? localeContent.Language : CultureInfo.InvariantCulture;
+                var contentName = contentData.Name?.Trim('-');
```

Comment:
Also have to remove or replace `--` in the name, given the error message `An XML comment cannot contain '--', and '-' cannot be the last character.`

Reply:
And perhaps use `TrimEnd`?

Reply:
You're correct, changed to use regex instead